src/c++/library/http_client.cc

```diff
@@ -951,6 +953,16 @@ InferResultHttp::IsFinalResponse(bool* is_final_response) const
   return Error::Success;
 }
 
+Error
+InferResultHttp::IsNullResponse(bool* is_null_response) const
```

Comment:
HTTP can't have a null response. Should this even have an implementation?

Reply:
i see.

Reply:
Meant to say false.